src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -283,6 +284,7 @@ public Object parse(@NotNull final MinecraftServer server, @NotNull final IComma
      * We are returning null to indicate that no boolean value could be parsed.
      */
     @SuppressWarnings({"squid:S2447"})
+    @Nullable
     private static Boolean parseBoolean(final String potentialArgumentValue)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this method 11, down to the maximum allowed 6. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)